gradle/scripts/lib/common-dependencies.gradle

```diff
@@ -159,6 +159,22 @@ configure(projectsWithFlags('java')) {
         }
     }
 
+    configurations.create("internal") {
```

Comment:
This looks similar to what we found in otel. You may consider naming it dependencyManagement since it shows in `:dependencies` task

https://github.com/open-telemetry/opentelemetry-java/blob/main/buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts#L191

Also btw I think those build scripts are generally pretty solid so crib what you can :)

Reply:
👍 `dependencyManagement` sounds better.